projects/storefrontlib/src/shared/components/select-timezone/timezones.ts

```diff
@@ -0,0 +1,1349 @@
+export const timezones = [
```

Comment:
This is a lot of additional code that will enlarge the final JS app shipped to each end user, so it slows down the user experience. Adding it should be justified. Where did we copied it from?

Could you explain why we need all this data? Ideally we could get rid of them. Or at least truncate unused properties.

Reply:
It was there to allow for timezone selection but we are no longer using it for this implementation. I thought to leave it there for use in the future but you are right. It does not have any purpose anymore and should be removed.